lib/puppet/face/module/list.rb

```diff
@@ -140,7 +140,7 @@ def warn_unmet_dependencies(environment)
             parent_version     = dep[:parent][:version]
 
             msg = "'#{parent_name}' (#{parent_version})"
-            msg << " requires '#{dep_name}' (#{version_constraint})"
+            msg << _(" requires '%{dep_name}' (%{version_constraint})") % { dep_name: dep_name, version_constraint: version_constraint }
```

Comment:
An you add a translator note that these two lines are concatenated to form one sentence w/example of what the output would be?